test/e2e/kubernetes/pod/pod.go

```diff
@@ -120,15 +120,15 @@ type Status struct {
 }
 
 // CreatePodFromFile will create a Pod from file with a name
-func CreatePodFromFile(filename, name, namespace string) (*Pod, error) {
+func CreatePodFromFile(filename, name, namespace string, sleep, duration time.Duration) (*Pod, error) {
```

Comment:
This change accommodates retry tolerance when `get`ing the pod object from k8s API